pyproject.toml

```diff
@@ -38,9 +16,56 @@ requires = [
     "typing-extensions>=4.10.0",
     "requests",
 ]
-# Use legacy backend to import local packages in setup.py
-build-backend = "setuptools.build_meta:__legacy__"
+build-backend = "setuptools.build_meta"
```

Comment:
Ultimately, this still builds under build isolation which is a big gotcha for the CI. Can we migrate to legacy scikit-build? That at least does a lot of stuff to effectively disable build isolation by default.

Reply:
Scikit-build-core works too, but I'm not sure how stable the setup tools shim is for it.